clients/fides-js/src/lib/cookie.ts

```diff
@@ -118,6 +118,12 @@ export const makeFidesCookie = (consent?: CookieKeyConsent): FidesCookie => {
   };
 };
 
+/**
+ * Retrieve cookie by name
+ */
+export const getCookieByName = (cookieName: string): string | undefined =>
+  getCookie(cookieName, CODEC);
```

Comment:
extracted this shared function so that we don't have to be aware of the `CODEC` across other files